File: src/main/java/de/steklopod/sec02/Lec05FluxRange.java
package de.steklopod.sec02;

import de.steklopod.courseutil.Util;
import reactor.core.publisher.Flux;

public class Lec05FluxRange {

    public static void main(String[] args) {

        Flux.range(3, 10)
                .log()
                .map(i -> Util.faker().name().fullName())
                .log()
                .subscribe(
                        Util.onNext()
                );


    }

}

File: src/main/kotlin/de/steklopod/sec02/Lec05FluxRange.kt
package de.steklopod.sec02

import de.steklopod.utils.Util.faker
import de.steklopod.utils.Util.onNext
import reactor.core.publisher.Flux

object Lec05FluxRange {
    @JvmStatic
    fun main(args: Array<String>) {
        Flux.range(3, 10)
            .log()
            .map { faker().name().fullName() }
            .log()
            .subscribe(
                onNext()
            )
    }
}
